dashboard/db/migrate/20210429195153_rename_misc_survey_to_simple_survey_submission.rb

```diff
@@ -0,0 +1,11 @@
+class RenameMiscSurveyToSimpleSurveySubmission < ActiveRecord::Migration[5.2]
+  def change
+    rename_table :foorm_misc_surveys, :foorm_simple_survey_submissions
```

Comment:
During release DB migrations are applied before servers are upgraded. Do we need to worry about issues coming up between those two events? (I had to write this COE because I ran into this in the past https://docs.google.com/document/d/1QHCjUdLz7D7fE-Cy4HrrtJ5FOSnSth7sNHfSemwNSfw/edit)

Reply:
Ooh that is a great question -- I do not think any of these surveys are actively in use, but I know James has this PR open to re-enable the NPS one:

https://github.com/code-dot-org/code-dot-org/pull/40262

I'll confer with him on timing and confirm that none of the other surveys are being actively used.